copy-app-conf.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+CONTEXT=context
+
+APP_INSTANCE="$CONTEXT/instance"
```

Comment:
```suggestion
APP_INSTANCE="context/instance"
```
In the original `$CONTEXT` is reused; Here it isn't.